packages/cssnano-preset-default/package.json

```diff
@@ -43,7 +43,8 @@
     "postcss-reduce-initial": "^4.0.3",
     "postcss-reduce-transforms": "^4.0.2",
     "postcss-svgo": "^4.0.2",
-    "postcss-unique-selectors": "^4.0.1"
+    "postcss-unique-selectors": "^4.0.1",
+    "postcss-unused-var": "^0.1.1"
```

Comment:
Please start plugin with `1.0.0` version

Reply:
ok I will publish v1 first and then I will change it here 👍